pkg/types/baremetal/platform.go

```diff
@@ -61,4 +61,14 @@ type Platform struct {
 
 	// DNSVIP is the VIP to use for internal DNS communication
 	DNSVIP string `json:"dnsVIP"`
+
+	// BootstrapMachineOSImage is a URL to override the default machine OS image
```

Comment:
`BootstrapMachineOSImage` `Machine` doesn't add any extra information, OSImage is clear enough i think.

Reply:
This can be renamed, but FWIW I think the naming here reflects some previous discussion with @smarterclayton who indicated the rhcos image references were all going to be replaced with "machineOS" to support FCOS builds.

Perhaps it's not neccesary to expose that naming convention in this interface however.

Reply:
@hardys I have no preference to the name. What is your choice?

Reply:
Took an executive decision here. Remove Machine from the name.